core/src/components/item-sliding/readme.md

```diff
@@ -692,6 +692,22 @@ Type: `Promise<number>`
 
 
 
+### `open(side?: string) => Promise<void>`
+
+Open the sliding item.
+
+#### Parameters
+
+| Name   | Type     | Description |
+| ------ | -------- | ----------- |
+| `side` | `string` |             |
```

Comment:
Can we document the description? It should be like the following in the code:

```
  /**
   * Open the sliding item.
   *
   * @param side The side of the options to open. Defaults to `"end"`.
   */
```

Or however you want to describe it. It could just be 

> If a side is not provided, it will open the options on the `"end"` side.

Reply:
Yep, I can update the PR with this change tonight.

Reply:
I just took care of it. Thank you though 🙂